app.grayjay.Desktop.yaml

```diff
@@ -0,0 +1,85 @@
+app-id: app.grayjay.Desktop
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+
+command: Grayjay
+finish-args:
+  - --share=network
+  - --socket=pulseaudio  # We want to play sound, I suppose
+  - --socket=x11
+  - --share=ipc # https://docs.flathub.org/docs/for-app-authors/linter#finish-args-x11-without-ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
+  - --own-name=org.mpris.MediaPlayer2.grayjay
```

Comment:
why do you need both names?

Reply:
This part of the config was based on what i saw in the FreeTube flatpak manifest. While freetube is based on electron, and this is based on CEF (Chrome embedded framework), I suspect that theres chromium in there somewhere. If needed i guess i could remove it and see if it causes any issues

Reply:
This is not freetube. One doesn't simply put random permissions because they read it elsewhere.

Reply:
This has been addressed by removing the grayjay specific name as a deeper dive with the [debugging tools](https://docs.flatpak.org/en/latest/debugging.html#audit-session-or-system-bus-traffic) only shows errors with one of these identifiers